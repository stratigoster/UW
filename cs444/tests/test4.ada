PACKAGE foo IS
  boo : STRING := "BOO";
  s : STRING := "Bye World""" & boo;
  b2 : BOOLEAN := s;
--                ^
-- incompatible types in assignment of string to boolean
END foo;

