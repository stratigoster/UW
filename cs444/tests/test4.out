***ERROR***:4:6:	Incompatible types in assignment of ` universal string ' to ` universal boolean '

------
Compilation error
------
